sycl/source/detail/global_handler.cpp

```diff
@@ -140,7 +140,12 @@ void shutdown() {
       // some parameters in the plugin tear-down process.
       // Currently, it is not used.
       void *PluginParameter = nullptr;
-      Plugin.call_nocheck<PiApiKind::piTearDown>(PluginParameter);
+      auto Error = Plugin.call_nocheck<PiApiKind::piTearDown>(PluginParameter);
+
+      if (Error != PI_SUCCESS) {
+        Plugin.reportPiError(Error, "piTearDown()");
+      }
```

Comment:
Why do we need `call_nocheck` here if we end up throwing an exception when smth different from `PI_SUCCESS` is returned? I think it's shorter to use `Plugin.call` here.

Reply:
good catch, got overlooked. I'll use `Plugin.call`

Reply:
done.

Reply:
done.